spec/models/organization_spec.rb

```diff
@@ -8,4 +8,43 @@
 
   it_should_behave_like 'notable'
 
+  it 'should set and get custom values' do
```

Comment:
Good specs save us a lot of time in the long run, not just on long-term maintenance but it also gives us an early warning during development before we introduce regressions that end up going unnoticed and becoming harder to correct later once more has changed around them. 

If `CustomValueSettable` is going to be used more than just in organizations, then it should be a shared example group. Same goes for `CustomModelLinkable`. These concerns usually only have one public method that you interface with, which takes a pretty simple argument. The shared example groups shouldn't really require more arguments than that, but you will need to write the tests in a more generic way. Things like "should be able to set custom values" and "can filter by custom values" and "should inherit custom field sets" are presumably all things that should work for *every* model that includes `CustomValueSettable` not just organizations, so having a shared example group saves us time re-implementing all of the same specs for those models.

I'm definitely sensitive to the project budget concerns, and if you don't think factoring these out as a shared group would be relatively quick & straightforward I'm not going to force the issue, but we should flag these with a `TODO:` in that case and try to get to it when we can, I think it's pretty important that we can test our concerns in a generalizable way.